docs/design_decisions.md

```diff
@@ -151,8 +151,9 @@ Binnen de historie-endpoints wordt alleen de actuele situatie met betrekking tot
 
 ### DD5.1 Descriptions worden als sibling van $Ref's opgenomen
 
-Waar in OAS3 een Description binnen een object naast een $ref niet toegestaan (daar is geen enkele sibling geoorloofd) is wordt deze in de draft van OAS3.1 wel toegestaan. Vooruitlopend op deze nieuwe versie is het binnen de HaalCentraal-specificaties geaccepteerd dat er binnen een object naast een $ref een Description wordt opgenomen. 
+Waar in OAS3 een Description binnen een object naast een $ref niet toegestaan is (daar is geen enkele sibling geoorloofd) wordt deze in de draft van OAS3.1 wel toegestaan. Vooruitlopend op deze nieuwe versie is het binnen de HaalCentraal-specificaties geaccepteerd dat er binnen een object naast een $ref een Description wordt opgenomen. 
 In OAS3 wordt er in deze gevallen een warning gegeven, aangezien dit soort descriptions echter binnen de geresolvde versie van het yaml bestand niet meer voorkomen wordt deze warning voor lief genomen. [Zie hier](https://github.com/OAI/OpenAPI-Specification/issues/2033) voor een toelichting.
+Om te zorgen dat de description in OAS3 toch wordt getoond wordt deze ook opgenomen in het schema component waar de $ref naar verwijst.
```

Comment:
niet in de schema component waar de $ref naar verwijst, maar in de schema component waarin de property zit met $ref

Reply:
Die begrijp ik niet goed. Het is toch juist de description die direct naast de '$ref' wordt geplaatst die voor de warning zorgt en die bij resolven ook wordt verwijderd. Om toch iets van een description te hebben moet je hem dus ergens anders plaatsen.

Dus aan dit fragment:

```
components:
  schemas:
    Beslag:
      type: object
      properties:
        identificatie:
          type: string
        aandeelInBetrokkenRecht:
          $ref: '#/components/schemas/TypeBreuk'
```

voeg je een 

```
          description: '..........'
```

toe voor de OAS3.1. Deze wordt echter verwijderd en daarom voeg je aan het component 'TypeBreuk' nog een 'description' toe, of niet?

Reply:
bedoel je wellicht dat er in dit geval aan 'Beslag' nog een 'description' wordt toegevoegd waarin je een beschrijving geeft van 'aandeelInBetrokkenRecht'?

Reply:
meestal gaat het zoals jij zegt. Dan staat in de component waarnaar verwezen wordt een description en die is specifiek genoeg. Daar gaat dit punt niet over, hier is er geen probleem met de descriptions.

Alleen wanneer je een generieke component gebruikt, is vaak niet meer duidelijk wat dit is.
Bijvoorbeeld gebruik van datum_onvolledig kan gebruikt worden voor een geboortedatum, een overlijdensdatum, een aanvangsdatum tenaamstelling, enz. Doe je niks, dan krijg je de description te zien die staat bij datum_onvolledig. Dat is iets generieks als "Gegevens over de datums die mogelijk niet volledig zijn". Maar soms wil je weten wat die datum is, dus waar gaat die datum over.
Ander voorbeeld is het gebruik van tabellen/waardelijsten. Die gebruik je voor allerlei soorten gegevens die verwijzen naar allerlei verschillende tabellen. De description in de component Waardetabel is te algemeen, geldt namelijk voor alle mogelijke tabellen. Bij het property die hiernaar verwijst moet je bijvoorbeeld wel melden welke waardetabel precies gebruikt wordt.
En in die situaties moet je dus in het component waarin het property zit de description opnemen. Bijvoorbeeld:
    KadastraalOnroerendeZaak:
      type: object
      properties:
        aardCultuurBebouwd:
          $ref: '#/components/schemas/Waardelijst'
        aardCultuurOnbebouwd:
          $ref: '#/components/schemas/Waardelijst'
      description: "Een kadastraal onroerende zaak is een perceel of een appartementsrecht.\
        \ Een perceel is een stuk grond waarvan het Kadaster de grenzen heeft gemeten\
        \ en dat bij het Kadaster een eigen nummer heeft. Een appartementsrecht is\
        \ het recht dat iemand of een organisatie heeft om eigenaar te zijn van een\
        \ deel van een gebouw of een stuk grond. In de praktijk wordt dit deel meestal\
        \ een appartement genoemd.\n\nWaardelijst in deze component :  [aardCultuurBebouwd](http://www.kadaster.nl/schemas/waardelijsten/CultuurcodeBebouwd/)\
        \ en [aardCultuurOnbebouwd](http://www.kadaster.nl/schemas/waardelijsten/CultuurcodeOnbebouwd/) "

Reply:
Duidelijk, ik heb de omschrijving aangepast.